tests/browser_e2e_tests/registration_tests.rs

```diff
@@ -0,0 +1,122 @@
+use crate::utils::browser::BrowserHelpers;
+use playwright::api::Page;
+use std::error::Error;
+
+pub async fn test_registration_flow(page: &Page) -> Result<(), Box<dyn Error>> {
+    // Step 1: Navigate to registration page
+    page.goto_builder("http://localhost:5435/register")
+        .timeout(5000.0)
+        .goto()
+        .await?;
+
+    // Verify page title
+    assert_eq!(page.title().await?, "Create account - ayb");
+
+    // Screenshot comparison of registration page
+    let registration_page_matches = BrowserHelpers::screenshot_compare(
+        &page,
+        "registration_page",
+        &[], // No elements to grey out for this test
+    )
+    .await?;
+
+    if !registration_page_matches {
+        println!("⚠ Registration page screenshot differs from reference");
+    } else {
+        println!("✓ Registration page matches reference screenshot");
+    }
```

Comment:
Push the if registration_page_matches logic into screenshot_compare. Remove the custom message and construct one from the image slug (i.e., `registration_page` in this case).